### ISSUE TITLE
Error "Cannot index into a null array" with psPAS 6.1.62 and IdentityCommand 0.2.46 to Privilege Cloud ServiceUser

### ISSUE THREAD

**Issue by mglanville2** [pspete/IdentityCommand#9]
Installed the new IdentityCommand version today and now cannot connect with psPAS to our Privilege Cloud tenant

**To Reproduce**
```powershell
PS P:\> Remove-Module psPAS
PS P:\> Import-Module IdentityCommand -RequiredVersion 0.2.46
PS P:\> Import-Module psPAS
PS P:\> New-PASSession -TenantSubdomain $tenant -Credential $credential -ServiceUser
Cannot index into a null array.
At line:789 char:6
 +...             $Script:WebSession.Headers['Authorization'] = [string]$Cy ...
 +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray 

PS P:\> Remove-Module psPAS
PS P:\> Import-Module IdentityCommand -RequiredVersion 0.1.43
PS P:\> Import-Module psPAS
PS P:\> New-PASSession -TenantSubdomain $tenant -Credential $credential -ServiceUser
PS P:\> get-passession

BaseURI                                                   ExternalVersion WebSession
-------                                                   --------------- ----------
https://ABC123.privilegecloud.cyberark.cloud/PasswordVault 0.0   
```

**Expected behavior**

Successful New-PASSession

**Screenshots & Console Output**

```powershell
 Cannot index into a null array.
At line:789 char:6
+ ...             $Script:WebSession.Headers['Authorization'] = [string]$Cy ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray 
```

**Your Environment**

* PowerShell Version: 5.1.19041.4046
* Module Version: IdentityCommand 0.2.46 with psPAS 6.1.62

**Additional context**

This tool saves me hours and hours of time, glad to help spot a problem.

**Comment by pspete** [comment]
Thanks for reporting @mglanville2 - you can uninstall 0.2.46 for now and continue using the previous version.
Will look into the cause and fix for a future release.

**Comment by pspete** [comment]
Should now be fixed in IdentityCommand v0.2.49.

Some required updates in `New-IDPlatformToken` had been overlooked - thanks again for reporting 👌
